src/screens/home/home.js

```diff
@@ -56,17 +64,17 @@ export default class Home extends React.Component {
 					<View style={ style.parkButton }>
 						<RoundButton
 							onPress={ () => this.handleClick() }
-							label={ 
-								this.state.currentParking 
-									? 'Deixar Vaga' 
-									: 'Estacionar' 
+							label={
+								this.state.currentParking
+									? 'Deixar Vaga'
+									: 'Estacionar'
 								}
 						/>
 					</View>
 					<Greetings/>
-					<Statistics/>
+					<Statistics {...this.state.stats}/>
```

Comment:
Só para seguir o padrão deixa` { ...this.state.stats }`

Reply:
Não muda nada, só pra falar alguma coisa mesmo hu3

Reply:
Beleza!